pytest.ini

```diff
@@ -0,0 +1,13 @@
+[pytest]
+addopts = 
+    # Kernel unavaiable.
+    --ignore=notebooks/2017-01-23-R-notebook.ipynb
+    --ignore=notebooks/2017-03-30-octave_notebook_example.ipynb
+    --ignore=notebooks/2017-08-01-xtractoR.ipynb
+    --ignore=notebooks/2017-11-30-rerddap.ipynb
+    --ignore=notebooks/2018-02-20-obis.ipynb
+    # Stale/won't fix.
+    --ignore=notebooks/2017-06-12-NCEI_RA_archive_history.ipynb
+    # Too slow.
+    --ignore=notebooks/2016-11-15-glider_data_example.ipynb
+    --ignore=notebooks/2017-07-25-HFRadar_currents.ipynb
```

Comment:
We are no longer testing R and octave kernels or long running notebooks.
There is also a list for stale/won't fix notebooks.